internal/provider/function_direxists_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package provider
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/config"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/knownvalue"
+	"github.com/hashicorp/terraform-plugin-testing/plancheck"
+)
+
+func TestDirectoryExists_basic(t *testing.T) {
+	resource.UnitTest(t, resource.TestCase{
+		// TerraformVersionChecks: []tfversion.TerraformVersionCheck{
+		// 	tfversion.RequireAbove(tfversion.Version1_8_0),
```

Comment:
Submitted: https://github.com/hashicorp/terraform-plugin-testing/pull/278

Reply:
Oh! And you can/should use [`tfversion.SkipBelow()`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/tfversion#SkipBelow) so CI can still run against all earlier Terraform versions.

Reply:
Thanks for submitting that. Also whoopsie 😆, SkipBelow is indeed what I wanted lol. Fixed ⚡